packages/yoastseo/src/scoring/assessments/inclusiveLanguage/configuration/disabilityAssessments.js

```diff
@@ -3,29 +3,26 @@ import {
 	potentiallyHarmfulCareful,
 	potentiallyHarmfulUnless,
 	harmfulPotentiallyNonInclusive,
+	alternative,
 } from "./feedbackStrings";
 import { isPrecededByException, isNotPrecededByException } from "../helpers/isPrecededByException";
 import { isNotFollowedByException } from "../helpers/isFollowedByException";
+import { isNotFollowedAndPrecededByException } from "../helpers/isFollowedAndPrecededByException";
 import { includesConsecutiveWords } from "../helpers/includesConsecutiveWords";
 import { SCORES } from "./scores";
 import notInclusiveWhenStandalone from "../helpers/notInclusiveWhenStandalone";
-import { all as toBeForms } from "../../../../languageProcessing/languages/en/config/internal/passiveVoiceAuxiliaries";
-import { flatMap } from "lodash-es";
+import disabilityRules from "./disabilityRulesData";
 import { sprintf } from "@wordpress/i18n";
 
+// Create feedback strings specific to disability assessments.
 const derogatory = "Avoid using <i>%1$s</i> as it is derogatory. Consider using an alternative, such as %2$s instead.";
 const generalizing = "Avoid using <i>%1$s</i> as it is generalizing. Consider using an alternative, such as %2$s instead.";
-
 const medicalCondition = harmfulPotentiallyNonInclusive +
 	" Unless you are referencing the specific medical condition, consider using another alternative to describe the trait or behavior, such as %2$s.";
 const potentiallyHarmfulTwoAlternatives = "Avoid using <i>%1$s</i> as it is potentially harmful. " +
 	"Consider using an alternative, such as %2$s when referring to someone's needs, or %3$s when referring to a person.";
-
-// Create a list of all possible combinations of a verb to be and a quantifier.
-const quantifiers = [ "so", "very", "a bit", "really", "pretty", "kind of" ];
-const toBeQuantifier = flatMap( toBeForms, verbToBe => flatMap( quantifiers, quantifier => `${verbToBe} ${quantifier}` ) );
-// A list of requirements: words that need to be in front of "OCD" for it to be non inclusive.
-const ocdRequirements =  toBeForms.concat( toBeQuantifier );
+// This string is used for phrases with 'crazy'. We don't want to mention the whole phrase in the feedback but only the non-inclusive word 'crazy'.
+const phrasesWithCrazyFeedback = [ "Avoid using <i>crazy</i> as it is potentially harmful.", alternative ].join( " " );
```

Comment:
You could have considered a similar solution as for "OCD" here, with a `sprintf`. But let's add that to https://github.com/Yoast/wordpress-seo/issues/19516 😄